app/javascript/components/CartList.tsx

```diff
@@ -0,0 +1,75 @@
+import { Slot } from "@radix-ui/react-slot";
+import * as React from "react";
+
+import { classNames } from "$app/utils/classNames";
+
+type BaseProps = {
+  children: React.ReactNode;
+  className?: string;
+  extra?: React.ReactNode;
+} & React.HTMLAttributes<HTMLElement>;
+
+export const CartList = ({ children, className, ...props }: BaseProps) => (
+  <div
+    role="list"
+    className={classNames(
+      "rounded-sm border border-border bg-background *:not-first:border-t *:not-first:border-border",
```

Comment:
Let's not use child selectors and instead apply these styles to the child elements directly.